### ISSUE TITLE
using search in administration page takes you out of the tab

### ISSUE THREAD

**Issue by KaraAJC** [wiliajc87/humblehearts#150]
as it stands, search works, but when you "submit your search" the entire page refreshes, putting you back at the "default tab" and making you click back into the tab you were searching from to get the results.

**Comment by wiliajc87** [comment]
Yeah, I think we would need to do an AJAX call on the search--if someone is particularly good at that? Honestly, I'm not. AJAX is one of those things I'd like to focus on in the future, but it would be pretty painful for me to do right now. :-/
